perf/benchmark/run_benchmark_job.sh

```diff
@@ -111,6 +111,36 @@ function setup_fortio_and_prometheus() {
     export FORTIO_SERVER_POD
 }
 
+function port_forward_envoy() {
+  kubectl -n "${NAMESPACE}" port-forward "${1}" 15000 &
+}
+
+function kill_envoy_port() {
+    kill "$(lsof -t -i:15000)"
+}
+
+function collect_envoy_info() {
+  POD_NAME=${1}
+  FILE_SUFFIX=${2}
+
+  kill_envoy_port
+  port_forward_envoy "${POD_NAME}"
+  ENVOY_DUMP_NAME="${POD_NAME}"_"${FILE_SUFFIX}".yaml
```

Comment:
this should be 
`"${POD_NAME}_${FILE_SUFFIX}.yaml"`

Reply:
done!